### PR TITLE
POC showing kubeflow on Tekton

### PR DESCRIPTION
**Pull request by chitrangpatel**
This PR shows a working POC of a Tekton Ppipeline
generated by a kubeflow DSL.

### CODE REVIEW

**Comment by jerop** [comment]
Thank you Chitrang!  🤗 

@mihaimaruseac @laurentsimon, Chitrang reverse-engineered the [Tekton Pipeline](https://github.com/google/model-transparency/blob/fb818c25e9b636858c7525017f5ae6c06e6cd869/slsa_for_models/gcp/pipeline.yml) into a Kubeflow Pipeline, which would be more familiar with ML engineers. Users can then compile the Kubeflow Pipeline into a Tekton PipelineRun, using [Kubeflow on Tekton](https://github.com/kubeflow/kfp-tekton). PTAL 🙏🏾

**Comment by jerop** [comment]
@chitrangpatel please update the [future work](https://github.com/google/model-transparency/blob/fb818c25e9b636858c7525017f5ae6c06e6cd869/slsa_for_models/gcp/README.md#kubeflow-on-tekton) item for adding kubeflow on tekton

File: slsa_for_models/kubeflow/model_transparency.yaml
Comment (jerop) [inline slsa_for_models/kubeflow/model_transparency.yaml:1]: I think this should be user-generated when executing the kubeflow pipeline, not checked into the source repo 🤔

**Review by mihaimaruseac — APPROVED** [review]
This looks good to me, thank you for the PR!

**Comment by mihaimaruseac** [comment]
Can you fix the lint issues, please?

**Comment by mihaimaruseac** [comment]
We still need the lint fixed and then rebasing the PR on main should make it meargeable

File: slsa_for_models/kubeflow/images/build_model/build.sh [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+         exit;;
+      r) # requirements path
+	 requirements=$OPTARG;;
+      w) # workingDir
+	 workingDir=$OPTARG;;
+      s) # source code path
+	 sourcePath=$OPTARG;;
+      d) # result path digest
+	 resultPathDigest=$OPTARG;;
+      m) # model name
```
Comment (mihaimaruseac) [inline slsa_for_models/kubeflow/images/build_model/build.sh:43]:
```suggestion
      m) # model name
```

File: slsa_for_models/kubeflow/images/clone/clone.sh [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      \?) # Invalid option
+         echo "Error: Invalid option"
+         exit;;
+   esac
+done
+
+echo "cloning $url into ${target}"
+git clone ${url} ${target}
+cd ${target}
+RESULT_SHA=$(git rev-parse HEAD)
```
Comment (mihaimaruseac) [inline slsa_for_models/kubeflow/images/clone/clone.sh:56]:
```suggestion
RESULT_SHA=$(git rev-parse HEAD)
```

**Comment by mihaimaruseac** [comment]
Thank you!

**Pull request merged**